core/src/main/java/com/yahoo/ycsb/measurements/OneMeasurementHistogram.java

```diff
@@ -36,11 +36,39 @@
 	public static final String BUCKETS="histogram.buckets";
 	public static final String BUCKETS_DEFAULT="1000";
 
+
+	/**
+	 * Specify the range of latencies to track in the histogram.
+	 */
 	int _buckets;
+
+	/**
+     * Groups operations in discrete blocks of 1ms width.
+     */
 	int[] histogram;
+	
+	/**
+	 * Counts all operations outside the histogram's range.
+	 */
 	int histogramoverflow;
+	
+	/**
+	 * The total number of reported operations.
+	 */
 	int operations;
-	long totallatency;
+	
+	/**
+	 * The sum of each latency measurement over all operations.
+	 * Calculated in ms.
+	 */
+	double totallatency;

```

Comment:
Totallatency was always cast to a double when consumed. I doubt that we'd ever overflow a long as this variable but I felt better with it as a double from the start instead of a late cast. There's no strict reason that it has to be a double and I could change it back easily enough.